lib/davinci_pas_test_kit/custom_groups/v2.0.1/notification/pas_subscription_notification_test.rb

```diff
@@ -1,41 +1,44 @@
+require_relative '../../../urls'
+
 module DaVinciPASTestKit
   module DaVinciPASV201
+    # TODO: none of file name, class name, or ID match
     class PasUpdateNotificationTest < Inferno::Test
+      include URLs
       id :prior_auth_claim_response_update_notification_validation
       title 'Server notifies the client that the pended claim was updated.'
       description %(
         This test validates that the server can notify the client that a final
         decision has been made about a pended claim so that the client can query
-        for the details. Currently, testers attest that the decision on the prior
-        authorization request has been finalized so that Inferno can check the rest of
-        the pended workflow. In the future, Inferno will validate that the server
-        can accept requests for a subscription and send notifications alerting the client
-        to the availability of updates to the prior authorization request (see details on this limitation
-        [here](https://github.com/inferno-framework/davinci-pas-test-kit/blob/main/lib/davinci_pas_test_kit/docs/server_suite_description_v201.md#testing-limitations)).
+        for the details. This test depends on prior successful subscription creation in
+        Test Group **1 Subscription Setup**.
       )
       verifies_requirements 'hl7.fhir.us.davinci-pas_2.0.1@141'
 
+      input :access_token,
+            title: 'Notification Access Token',
+            description: %(
+              An access token that the server under test will send to Inferno on notifications
+              so that the request gets associated with this test session. The token must be
+              provided as a `Bearer` token in the `Authorization` header of HTTP requests
+              sent to Inferno.
+            )
+
       run do
```

Comment:
This test should only be run if there was a pended submission, which depends on the **PAS Submit Request Payload for Pended Response** input. I think ideally :subscriptions_r4_server_notification_conformance and :subscriptions_r4_server_id_only_conformance would as well - you could try adding that input to them so that it will skip if it isn't present.

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/pull/31/commits/2d199e6275948a43956eafbc202874b225d538a5